enterprise/views.py

```diff
@@ -996,6 +999,37 @@ def form_valid(self, form):
         )
 
 
+class EnterpriseProxyLoginView(View):
+    """
+    Allows an enterprise learner to login via existing flow from the learner portal.
+    """
+
+    def get(self, request):
+        """
+        Redirects a learner through login with their enterprise's third party auth if it uses tpa.
+        """
+        query_params = request.GET
+        redirect_to = query_params.get('next') or LMS_LOGIN_URL
```

Comment:
I'm not completely sure what the rationale on having this next parameter is, as it seems like it should always direct to LMS login? For now I've left it in as the ticket and starting implementation suggested, but I can remove it if others don't think there is a need for it.
@brittneyexline you might know the most here

Reply:
Although this might have been meant to just be a new next parameter for the login redirect, and the starting implementation mixed that up